_ml-commons-plugin/cluster-settings.md

```diff
@@ -59,7 +59,7 @@ plugins.ml_commons.max_ml_task_per_node: 10
 
 ## Set number of ML models per node
 
-Sets the number of ML models that can be loaded on to each ML node. When set to `0`, no ML models can load on any node.
+Sets the number of ML models that can be deployed on to each ML node. When set to `0`, no ML models can deploy on any node.
```

Comment:
deploy into, on to a node. Do these have different meanings?

Reply:
I think it should be "deploy to".